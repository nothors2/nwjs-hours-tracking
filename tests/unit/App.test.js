import { shallowMount } from '@vue/test-utils';
import App from '@/App.vue';

describe('App.vue', () => {
  test('Render default contents', () => {
    const wrapper = shallowMount(App);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
