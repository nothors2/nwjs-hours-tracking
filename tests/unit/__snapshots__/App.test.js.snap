// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue Render default contents 1`] = `
<div id="app">
  <img src="@/assets/vue.png" alt="Vue.js logo" title="Vue.js" class="logo">
  <img src="@/assets/nw.png" alt="NW.js logo" title="NW.js" class="logo">
  <helloworld-stub msg="Welcome to your Vue.js Desktop App in NW.js!"></helloworld-stub>
</div>
`;
